### ISSUE TITLE
install error on windows OS

### ISSUE THREAD

**Issue by stemicha** [STATWORX/helfRlein#22]
First of all, THANKS for your time and efforts in producing such a nice package.
If you want to install helfRlein on Windows OS you often run into an install error.

On some installation instances the installation worked with a warning.
> Warning: unable to re-encode 'char_replace.R' lines 79, 80

But on others it resulted in an error and I was not able to install it, not with binaries and not by using install_github.
 
Editing char_replace.R and remove line 79, 80 for installation on windows solved that issue.

Do you have a better workaround?

**Comment by Dschaykib** [comment]
Thanks for the feedback!
Since I am developing it on a Mac I did not have this issue yet, but I will have a look into that.

**Comment by Dschaykib** [comment]
Hey, I finally had time to try it on windows. I got the same warning and the problems lies with the different OS coding. Some of the char ar not accessible for all OS (like the "ﬁ" and "ﬂ" in those lines).

I found [this stackoverflow post](https://stackoverflow.com/questions/10294284/remove-all-special-characters-from-a-string-in-r) which looked promising:
```
x <- c("ﬁ", "ﬂ")
iconv(x, from = '', to = 'ASCII//TRANSLIT')
```
```
[1] "i??" "i?,"
```
But, this would not transform the characters like we wanted them to.
If you have an idea on how to fix this other then to remove those lines, it would be welcome!
I will further think about this.
